nextest.toml

```diff
@@ -16,6 +16,14 @@ path = "junit.xml"
 filter = "package(solana-zk-elgamal-proof-program-tests) & test(/^test_batched_range_proof_u256$/)"
 threads-required = "num-cpus"
 
+[[profile.ci.overrides]]
+filter = "package(solana-turbine) | package(solana-gossip)"
+retries = 0
+
+[[profile.ci.overrides]]
+filter = "package(solana-gossip) & test(/^test_star_network_push_star_200/)"
+threads-required = "num-cpus"
+
```

Comment:
if I'm not missing anything, I think these lines are the same as 27-30 🤔

Reply:
oops, fixed, apologies!